Ithenticate.php

```diff
@@ -216,6 +222,7 @@ public function createFolder($folder_name, $folder_description, $group_id, $excl
             'name' => new Value($folder_name),
             'description' => new Value($folder_description),
             'exclude_quotes' => new Value($exclude_quotes),
+            'add_to_index' => new Value($add_to_index),
```

Comment:
There were other parameters available for this method, but I just added what I needed :)